{{cookiecutter.project_name}}/tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+"""Global fixtures for {{cookiecutter.friendly_name}} integration."""
+import pytest
+from pytest_homeassistant_custom_component.async_mock import patch
```

Comment:
With latest version of `pytest-homeassistant-custom-component`, `async_mock` module doesn't exist anymore. It's due to homeassistant supporting only Python version > 3.8 so unittest can be used here.
```suggestion
"""Global fixtures for {{cookiecutter.friendly_name}} integration."""
import pytest
from unittest.mock import patch
```
Another reason to pin the version of `pytest-homeassistant-custom-component` ;-)